data/json/recipes/other/tool.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "result": "still",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "skills_required": [ "cooking", 3 ],
+    "difficulty": 4,
+    "time": 200000,
+    "book_learn": [ [ "textbook_chemistry", 4 ], [ "brewing_cookbook", 3 ] ],
+    "qualities" : [
+      { "id":"CUT", "level": 1, "amount": 1 },
+      { "id":"HAMMER", "level": 2, "amount": 1 }
+    ],
+    "components": [
+      [ [ "pot_copper", 1 ] ],
```

Comment:
Yes. It limits you to denatured alcohol until you can construct one. 'alcohol_denat` is a good substitute for almost all early game tasks with a future PR intending to make the higher end recipes require the purer `chem_ethanol`